src/screens/Forum/Forum.js

```diff
@@ -1,37 +1,42 @@
-import React, { useEffect } from 'react'
-import firebaseApp from '../../firebase'
+import React, { useEffect, useState } from 'react'
 import firebase from '../../firebase'
 
-import { CreateItem, ItemsList } from "components";
-
+import { ItemsList, CreateItem } from "components";
 import styles from './Forum.module.css'
 
 export function Forum() {
-  // const newPost = { title:"new title", question:"new question"}
-  // useEffect(()=>{
-  //   firebaseApp.db.collection('forum').add(newPost);
+  const [clicked, setClicked] = useState(false)
+
+
+  const addQuestion = () => {
 
-  // },[])
-   const addQuestion = () => {
-    // const userId = firebase.auth().currentUser.uid;
-   
     firebase.database().ref('forum').push({
-      user:'unknown',
-      title:'title',
-      question:'question',
+      user: 'unknown',
+      title: 'title',
+      question: 'question',
       createdAt: new Date().toISOString()
     })
- 
+
   };
 
-  useEffect(()=>{
-    console.log("DODAŁEM")
-  }, [])
-  return(
+  return (
     <div className={styles.mainDiv}>
       <div className={styles.content}>
-      <CreateItem addQuestion={addQuestion}/>
-      <ItemsList />
+        <h1>FORUM</h1>
+        {clicked ||
+          <h1>Do you wanna create a topic/ask question?
+            <a
+              style={{ cursor: "pointer", color: "black" }}
+              onClick={(e) => {
+                e.preventDefault()
+                setClicked(!clicked)
+              }}
+            >
+              CLICK HERE
+            </a>
+          </h1>}
+        <CreateItem addQuestion={addQuestion} clicked={clicked} />
+        <ItemsList />
       </div>
     </div>
   )
```

Comment:
After adding a question, the form doesnt hide neither there is no button to hide this form.